modules/ztp-factory-provisioning-clusters.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="factory-installing-edge-sites_{context}"]
+= Factory installing edge sites
+
+Zero touch provisioning (ZTP) deploys the {product-titler} spoke cluster architecture from a preinstalled OpenShift Container Platform (hub cluster) using RHACM (Red Hat Advanced Cluster Management).
+
+The "factory" high level architecture is illustrated in this diagram:
+
+image::network_topology.png[Network topology]
+
+Every blade in the chassis has access to multiple NICs, which are connected to internal switches. The switches and NICs are referred to as networks using the name of the interface.
+
+The eno4 and eno5 networks, are 10gbs networks with enough bandwidth to support the internal and external traffic of the cluster. Both networks are L2.
```

Comment:
The eno4 and eno5 are Network Interface Cards.
Don't think we should mention L2. What is the other possible option here?

Reply:
OK fair point